### ISSUE TITLE
Try out having all assets in the webpack JS

### ISSUE THREAD

**Issue by mliebelt** [mliebelt/PgnViewerJS#170]
Configuring the assets is a pain, and not a good solution. How much does it cost to have all assets as part of the JS file? How about having a builder (like a configurable call interface) to produce a package in naming the piece sets one by one?

At least I should know what the overhead is, so we can decide then on a known base how to produce in the future a package.

**Comment by mliebelt** [comment]
After having merged the changes of #187 locally, I was able to do some experiments with it:

* Old
** `npm build; npm pack`: `pgnv.js` is ~270,6 kB, asset directory has 281,0 kB.
** Tried to install it by having created a new project, doing there: `npm i @mliebelt/pgn-viewer`, and was astonished that the resulting `pgnv.js` file delivered there was 2,5 MiB (2.658.979 bytes)!! I have to create a ticket for that ... This file seems to include the sourcemap, like the development version of 
* New
** `npm build; npm pack`: `pgnv.js` is ~641,7 kB, no assets.
** I was able to use all the examples, everything worked :-)
** I I have to check if packing the file by using [CompressionWebpackPlugin](https://v4.webpack.js.org/plugins/compression-webpack-plugin/) could be an option.
